builtin/credential/approle/path_role.go

```diff
@@ -227,52 +227,6 @@ can only be set during role creation and once set, it can't be reset later.`,
 								Required:    true,
 								Description: "If true, the secret identifiers generated using this role will be cluster local. This can only be set during role creation and once set, it can't be reset later",
 							},
-							"token_bound_cidrs": {
```

Comment:
Ah, okay, I was going to comment about not wanting to make a breaking change by removing these fields, but it looks like we add them back later. I had to expand the top part to see that these were a part of the Fields field in the response, but it looks like it checks out!